fourier-polygon.js

```diff
@@ -23,7 +23,11 @@ Complex.prototype.mul = function (other) {
     this.re = re;
     this.im = im;
     return this;
-};
+	};
+Complex.prototype.div = function (other) {
+	return this.mul(1/other);
+	};
```

Comment:
This won't work. `mul` expects a complex, not a number. Regardless, this file should be identical to the other. This file will end up on the public facing webpage.